source/docs/robot-design/odometry.rst

```diff
@@ -100,8 +181,8 @@ the field.
 
 Spring Tensioning
 -----------------
-Spring tensioning is important for making sure there
-isn't a lot of slipping or skidding of the pods/odometers.
+Spring tensioning is important in reducing slipping or skidding of
+the pods/odometers.
```

Comment:
Maybe expand on what spring tensioning means for odometry?

Reply:
Ngl wasn't sure how to expand upon jackson's original explanation so I just left it. I think it would best be explained with sample images.

Reply:
"For odometry, spring tensioning is the use of a spring to push odometry modules towards the ground, either on a pivot (recommended) or linearly. Spring tensioning is important in reducing slipping or skidding of the pods/odometers." or something